lms/djangoapps/courseware/views/views.py

```diff
@@ -903,128 +950,80 @@ def _progress(request, course_key, student_id):
     return response
 
 
-def _get_cert_data(student, course, course_key, is_active, enrollment_mode, grade_summary=None):
+def _downloadable_cert_data(download_url=None, cert_web_view_url=None):
+    return CertData(
```

Comment:
Suggestion:  I like the constants created above.  To keep these consolidated in one place, consider moving this helper function up above.